cmd/influxd/run/server.go

```diff
@@ -160,6 +166,11 @@ func (s *Server) Open() error {
 			return fmt.Errorf("open tsdb store: %s", err)
 		}
 
+		// Open the hinted handoff service
```

Comment:
Because it's required by `PointsWriter` which all the other services depend on as well.  I didn't want to let `PointsWriter` get created without the hinted handoff service not set on it.  I may change this around in a future PR because `Enabled` is handled a little differently in hinted handoff right now.